src/cdk-experimental/listbox/listbox.ts

```diff
@@ -284,6 +304,16 @@ export class CdkListbox<T> implements AfterContentInit, OnDestroy, OnInit, Contr
     this._destroyed.complete();
   }
 
+  private _registerWithPanel(): void {
+    if (this._parentPanel === null || this._parentPanel === undefined) {
+      if (this._explicitPanel !== null && this._explicitPanel !== undefined) {
+        this._explicitPanel._registerContent(this.id, 'listbox');
+      }
+    } else {
+      this._parentPanel._registerContent(this.id, 'listbox');
+    }
```

Comment:
```suggestion
    const panel = this._parentPanel || this._explicitPanel;
    panel?._registerContent(this.id, 'listbox');
```
Similar for `_updatePanel` below

Reply:
Okay that's simpler.